pkg/volume/csi/nodeinfomanager/nodeinfomanager.go

```diff
@@ -378,8 +382,15 @@ func (nim *nodeInfoManager) tryUpdateCSINode(
 	driverNodeID string,
 	maxAttachLimit int64,
 	topology map[string]string) error {
-
-	nodeInfo, err := csiKubeClient.StorageV1beta1().CSINodes().Get(string(nim.nodeName), metav1.GetOptions{})
+	var nodeInfo *storagev1beta1.CSINode
+	var err error
+	if nim.nodeLister != nil {
+		nodeInfo, err = nim.nodeLister.Get(string(nim.nodeName))
+		klog.V(4).Infof("informer returned %v err: %v", nodeInfo, err)
+	} else {
```

Comment:
We should return error. If the nodeLister is nil, then that means there are bigger API issues, and falling back to `Get` would not help.